meson_options.txt

```diff
@@ -44,6 +44,7 @@ option('efi-ldsdir', type : 'string', description : 'path to the EFI lds directo
 option('efi-includedir', type : 'string', value : '/usr/include/efi', description : 'path to the EFI header directory')
 option('efi_os_dir', type: 'string', description : 'the name of OS directory in ESP, e.g. fedora')
 option('efi_sbat_fwupd_generation', type : 'integer', value : 0, description : 'SBAT fwupd generation')
+option('efi_sbat_distro_id', type : 'string', value : '', description : 'SBAT distribution ID, e.g. fedora')
```

Comment:
I think both `*os_dir`  values need the word `hardcoded` adjective added to them now to make it clearer why you would use them

Reply:
I'm not super keen changing the meson define name as that's going to break (silently?) distros using the old name to override the default -- or do you mean the description?

Reply:
Just the description field I mean.

Reply:
Ohh, something like `the hardcoded name of OS directory in ESP, e.g. fedora`?

Reply:
Exactly

Reply:
Fixed.